apps/web/app/(routes)/page.tsx

```
@@ -11,19 +10,19 @@ import { ServerType } from "@iflow/types";
 
   const API_ENDPOINT = process.env.NODE_ENV == "development" ? "https://api.indexflow.site/v1/bot/server/all" : "https://api.indexflow.site/v1/bot/server/all";
   // const API_ENDPOINT = process.env.NODE_ENV == "development" ? "http://localhost:3001/v1/bot/server" : "https://api.indexflow.site/v1/bot/server";
 
   useEffect(() => {
     async function fetchServers() {
-      setLoading(true);
+      setSkeleton(true);
 
       const data = await fetch(API_ENDPOINT);
       const response = await data.json();
       setServers([...response?.servers, ...servers]);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)